<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamingParametersConfigurationWindow</class>
 <widget class="QWidget" name="StreamingParametersConfigurationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Streaming Parameters</string>
  </property>
  <widget class="QLabel" name="videoParametersLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>150</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Video parameters:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>23</y>
     <width>400</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="videoBitrateLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>40</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Video bitrate:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="videoBitrateLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="videoBitrateMetricLabel">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kb / Frame</string>
   </property>
  </widget>
  <widget class="QLabel" name="videoSizeLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>70</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Video size:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="videoBitrateComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>380p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>420p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>720p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1080p</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="audioParametersLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>105</y>
     <width>150</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Audio parameters:</string>
   </property>
  </widget>
  <widget class="Line" name="line2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>115</y>
     <width>400</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="audioBitrateLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>135</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Audio bitrate:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="audioBitrateComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>135</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>56 kbps - Suitable for voice recordings</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>128 kbps - FM quality</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>256 kbps - Medium quality</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>320 kbps - CD quality</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitPushButton">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="Line" name="line3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>400</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="stereoRadioButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>165</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Stereo</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="monoRadioButton">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>165</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mono</string>
   </property>
  </widget>
  <widget class="QLabel" name="audioStereoMonoLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>165</y>
     <width>100</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Audio Stereo / Mono:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
